workers/analyzer/docker/Analyzer.Dockerfile

```diff
@@ -485,6 +485,8 @@ ENV CARGO_HOME=$RUST_HOME/cargo
 ENV RUSTUP_HOME=$RUST_HOME/rustup
 ENV PATH=$PATH:$CARGO_HOME/bin:$RUSTUP_HOME/bin
 COPY --from=rust --chown=$USER:$USER /opt/rust /opt/rust
+RUN $CARGO_HOME/bin/cargo install cargo-credential-netrc
+RUN chown -R $USER:$USER /opt/rust/cargo/registry/cache
```

Comment:
Is this maybe what's missing for the (still failing) [ORT PR](https://github.com/oss-review-toolkit/ort/pull/10808)?

Reply:
Oh, I just saw [you changed it](https://github.com/oss-review-toolkit/ort/compare/3ecf8f667ba188208532261c06f9972eca076692..79871fc65ea8158ed0bca3e5bd176244799a459e).

Reply:
No, the ORT PR has (sort of) the same code but fails : https://github.com/oss-review-toolkit/ort/pull/10808/files

I really don't know what going on there, as

```
docker run --rm  -u $(id -u):$(id -g)  -v /etc/group:/etc/group:ro  -v /etc/passwd:/etc/passwd:ro -v .:/workspace -v /tmp/:/home/runner -w /workspace -e HOME=/home/runner -e GIT_CONFIG_NOSYSTEM=true -e GRADLE_USER_HOME=/tmp -e HOME=/home/runner -e GRADLE_OPTS="-Dorg.gradle.daemon=false -Dkotest.assertions.multi-line-diff=unified -Dkotest.tags=\!ExpensiveTag" ort:test -c "./gradlew :plugins:package-managers:cargo-package-manager:funTest"
```

pass for me locally, and should be exactly what the CI check does.